scripts/prep-release.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Helper script to prepare a release for the Go SDK.
+
+# Read the version number and build number from the respective files
+current_version_number=$(< internal/version)
+current_build_number=$(< internal/version-build)
```

Comment:
these two shouldn't be required anymore, correct?

Reply:
The current_version_number isn't required but the current_build_number is as later on in the script, we test to see if the build # has changed as we don't want the user to try to make a release with the same version.